downgrade

```diff
@@ -272,7 +353,9 @@ cli() {
     }
   fi
 
-  main "${terms[@]}"
+  for term in "${terms[@]}"; do
+    main "$term" || (gettext "No package with name $term found" && echo && gettext "$(usage)" && exit 1)
+  done
```

Comment:
I'd rather not print `usage` on this error. It's not invalid usage IMO, so the help text would just be noise that could obscure the real error.

Also, this is a behavior change. Currently, `downgrade` will try all arguments, and exit `1` if any fail. This loop now exits on any failure immediately. I'm not necessarily against that, but you're going to want to remove the failure-handling in `main` if that's the behavior you're after.

Personally, I like it the way it is, and you could add a `gettext` error message within the `main` loop, if you just want some more user feedback.

Reply:
Agree with that. I added the `gettext` for user feedback when the package name supplied does not exist. This is now directly in `main`. However, this leads to an error with in tests `test/main/invalid-choice.t` and `test/main/no-candidates.t` because this prints text where it was not previously expected. Should the test script be modified for this case?